01_variables/a_dumb_calculator.py

```diff
@@ -3,4 +3,16 @@
 # Then add the numbers together and divide by 2 
 # Display the result 
 # Your program must support numbers with decimals and not just integers 
-# for more info on this quiz, go to this url: http://www.programmr.com/dumb-calculator-5
\ No newline at end of file
+# for more info on this quiz, go to this url: http://www.programmr.com/dumb-calculator-5
+
+
+def simple_calc():
+    numbers = input("Enter three numbers separated with a space: ")
+    lst = numbers.split()
+    total = 0
+    for i in lst:
```

Comment:
Another way to do this instead of looping over the list is to use pythons list method sum
i.e 
```python 
sum([1, 2, 3, 4]) # 10
```